### PR TITLE
DOCS-12115 update kubernetes version

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

**Review by sathya0 — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -163,6 +163,11 @@ Example output plugin configuration:
 
 ==== Elasticsearch Configuration Parameters
 
+[IMPORTANT]
+--
+Runtime Fabric 1.10.26 and later uses Fluent Bit 1.83 for external log forwarding. If you forward logs to Elasticsearch, you must set the  configuration parameter `Generate_ID` to `On`. By default, Fluent Bit sets this parameter to `Off`.
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]:
```suggestion
Runtime Fabric 1.10.26 and later uses Fluent Bit 1.8.3 for external log forwarding. If you forward logs to Elasticsearch, you must set the  configuration parameter `Generate_ID` to `On`. By default, Fluent Bit sets this parameter to `Off`.
```

**Comment by vli63127** [comment]
lgtm

**Review by sarjeet2013 — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 2]
```diff
@@ -163,6 +163,11 @@ Example output plugin configuration:
 
 ==== Elasticsearch Configuration Parameters
 
+[IMPORTANT]
+--
+Runtime Fabric 1.10.26 and later uses Fluent Bit 1.8.3 for external log forwarding. If you forward logs to Elasticsearch, you must set the  configuration parameter `Generate_ID` to `On`. By default, Fluent Bit sets this parameter to `Off`.
```
Comment (dukesphere) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]:
```suggestion
Runtime Fabric 1.10.26 and later uses Fluent Bit 1.8.3 for external log forwarding. If you forward logs to Elasticsearch, you must set the configuration parameter `Generate_ID` to `On`. By default, Fluent Bit sets this parameter to `Off`.
```

**Review by dukesphere — DISMISSED** [review]
small suggestion to remove extra space

**Review by dukesphere — APPROVED** [review]

**Pull request merged**